src/theme/components/template.ts

```diff
@@ -91,9 +92,11 @@ const TemplateContentSidebar = defineStyleConfig({
     left: {
       gridColumn: "1",
       paddingLeft: "s",
+      paddingRight: { base: "s", md: undefined },
```

Comment:
With this addition, the spacing is now off in the desktop view. I think you'll need to set `md: "0"` instead of `undefined` or `null`.

Reply:
Thank you! Yes, that's right.